bootloader/src/pyi_main.c

```diff
@@ -21,8 +21,10 @@
 #ifdef _WIN32
     #include <windows.h>
     #include <wchar.h>
+    #include "old_machine_bootloader_windows.h"
 #else
     #include <limits.h>  /* PATH_MAX */
+    #include "old_machine_bootloader_linux.h"
 #endif
```

Comment:
Can simplify

Reply:
How?

Reply:
You can include a single header file and have the C file be guarded with an IFDEF (I think you already have that)

Reply:
you mean to include a single `old_bootloader.h` and merge all windows and linux code into `old_bootloader.c`?

Reply:
The best solution is
* A single public header file for both OSes. If required, with IFDEF guards but I don't think that's required.
* Two .c files for OS specific functionality.
* A .C file with shared functionality, if relevant. If required, it'll also include a shared header file.